metadata.json

```diff
@@ -74,11 +75,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": "4.x"
-    },
-    {
-      "name": "pe",
-      "version_requirement": ">= 2016.2.0"
+      "version_requirement": ">= 4.6"
```

Comment:
* Is there a reason why this doesn't also define an [upper boundary](https://docs.puppet.com/puppet/4.9/modules_metadata.html#best-practice-set-an-upper-bound-for-dependencies)?
  * "`4.x`" also defined an upper limit.
  * "`>= 4.6.0 < 5.0.0`" should work.


* "`4.6`" should be "`4.6.0`".  

  * The [documented version specifier format](https://docs.puppet.com/puppet/4.8/modules_metadata.html#version-specifiers) appears to only support three-part specific versions (`MAJOR.MINOR.PATCH`) and two-part ranges using the _x_ character as a wildcard (`MAJOR.x `)   

  * The validity of two-part (`MAJOR.MINOR`) versions isn't specifically documented, but the comments for [the relevant unit tests](https://github.com/puppetlabs/puppet/blob/4.8.2/lib/puppet/vendor/semantic/spec/unit/semantic/version_spec.rb#L28-L30) in 4.8.2 indicate that this format should not be used.